libpacaur/pkgs.sh

```diff
@@ -300,7 +305,7 @@ MakePkgs() {
         # build
         Note "i" $"Building ${colorW}${pkgsdeps[$i]}${reset} package(s)..."
 
-        # install then remove binary deps
+        # build then remove binary deps
```

Comment:
Actually, the comment is about the whole `if then` code block below. `makeopts=(${makeopts[@]/-r/})` is done to unset the `-r` flag to ensure there won't be any issue with the next package to build.

That line might be moved inside the `if then` block, I'll have to check why I set it outside.